R/annotateProteinInfoFromIndra.R

```diff
@@ -0,0 +1,115 @@
+#' Annotate Protein Information from INDRA
+#'
+#' This function annotates a given dataframe with additional protein information 
+#' using various APIs. The input dataframe must contain columns 'Protein' and 'IDType'.
+#' The function adds new columns to the dataframe, including 'UniprotID', 'HgncID', 
+#' 'HgncName', 'IsKinase', 'IsPhosphatase', and 'IsTranscriptionFactor'.
+#'
+#' @param df A dataframe containing at least 'Protein' and 'IDType' columns.
+#' @return A dataframe with additional columns: 'UniprotID', 'HgncID', 'HgncName', 
```

Comment:
Can you describe the collumns of the output data frame like how I do it here:

https://github.com/Vitek-Lab/MSstats/pull/142/files#diff-78b6c05b99c2091421cda250299bc5c6dc31033d6db7787a48859a20aeabe8daR21-R25

Reply:
done